app/driver/settings_travis.py

```diff
@@ -0,0 +1,147 @@
+
+"""
+Django settings for DRIVER project testing.
```

Comment:
Is there a better way to override this file? All that is different is the DATABASES key.

If there is an easier way using the base settings.py file I don't have secrets.yml file either. So that would have to be stubbed out somehow.